simpleast-core/src/main/java/com/discord/simpleast/core/simple/SimpleMarkdownRules.kt

```diff
@@ -11,19 +11,18 @@ import com.discord.simpleast.core.node.TextNode
 import com.discord.simpleast.core.parser.ParseSpec
 import com.discord.simpleast.core.parser.Parser
 import com.discord.simpleast.core.parser.Rule
-import java.util.*
 import java.util.regex.Matcher
 import java.util.regex.Pattern
 
 object SimpleMarkdownRules {
 
-  val PATTERN_BOLD = Pattern.compile("^\\*\\*([\\s\\S]+?)\\*\\*(?!\\*)")
-  val PATTERN_UNDERLINE = Pattern.compile("^__([\\s\\S]+?)__(?!_)")
-  val PATTERN_STRIKETHRU = Pattern.compile("^~~(?=\\S)([\\s\\S]*?\\S)~~")
-  val PATTERN_TEXT = Pattern.compile("^[\\s\\S]+?(?=[^0-9A-Za-z\\s\\u00c0-\\uffff]|\\n\\n| {2,}\\n|\\w+:\\S|$)")
-  val PATTERN_ESCAPE = Pattern.compile("^\\\\([^0-9A-Za-z\\s])")
+  private val PATTERN_BOLD = Pattern.compile("^\\*\\*([\\s\\S]+?)\\*\\*(?!\\*)")
```

Comment:
These fields should remain public, they are designed to be used in client apps.

Reply:
got it. updated. :)